recipes-ofdpa/ofdpa/ofdpa_3.0.5.0-EA5.bb

```diff
@@ -1,7 +1,7 @@
 DESCRIPTION = ""
 LICENSE = "CLOSED"
 
-PR = "r41"
+PR = "r42"
 SDK_VERSION = "6.5.24"
 SRCREV_ofdpa = "0950c31e9629a83b916f6f138a3dd9c6899debb1"
```

Comment:
If we bump the PR, I think this needs to be updated as well.

Reply:
This is the revision of the OF-DPA source code repository, not the internal meta-ofdpa repository. There were no changes there, and 0950c31e9629a83b916f6f138a3dd9c6899debb1 is still HEAD of main.

If you look at the internal commit, you'll see it didn't change there either.